qa/integration-tests/src/test/java/io/camunda/it/client/UserTaskQueryTest.java

```diff
@@ -368,10 +367,12 @@ public void shouldRetrieveTaskByCandidateGroup() {
   @Test
   public void shouldRetrieveTaskByCandidateGroupFilter() {
     // when
-    final var filter = new StringFilterProperty();
-    filter.$like("grou?");
     final var result =
-        camundaClient.newUserTaskQuery().filter(f -> f.candidateGroup("group")).send().join();
+        camundaClient
+            .newUserTaskQuery()
+            .filter(f -> f.candidateGroup(b -> b.like("grou?")))
+            .send()
+            .join();
```

Comment:
❓ Should we add a couple more ITs that ensure more edge cases work as expected e2e, like escaping things and using characters that would normally be interpreted as wildcards in SQL? I know we have unit tests that test they are transformed as expected but I think it makes sense to also ensure e2e they are executable and lead to the results we expect in every data layer.

Reply:
I was also thinking about this. I'll add a couple tests to see that everything is interpreted in all dbs properly.